Add  Bit gold  logo

sadjkkgi opened the pull request:
30000TPT:0xa47ef0d49bfd7b6eb4ca029754022e64d8283aadb662c5e867f56f5a7fe48039
validation：0x3741349f9fd414b9d4705e17497104d3731661fdd13dee6f6b2f19df5cbf1be9

opera0001 commented:
It is recommended that you replace the logo. This logo is very similar to the logo circulated on the Internet. It is uncertain whether there will be infringement. In order to avoid unnecessary trouble, it is best to replace it with a new logo scheme. thank you.

sadjkkgi commented:
> 建议您更换徽标。这个标志与互联网上流传的标志非常相似。是否会侵权尚不确定。为了避免不必要的麻烦，最好换一个新的logo方案。谢谢。

Ok! I have modified and replaced it, please help me show it as soon as possible! Thank you very much!
